packages/react-dev-utils/printFileSizes.js

```diff
@@ -0,0 +1,48 @@
+var fs = require('fs');
+var path = require('path');
+var chalk = require('chalk');
+var filesize = require('filesize');
+var stripAnsi = require('strip-ansi');
+var gzipSize = require('gzip-size').sync;
+var removeFileNameHash = require('./removeFileNameHash');
+var getDifferenceLabel = require('./getDifferenceLabel');
```

Comment:
I don't think these should be in separate files, please move these functions in this file (but not inside the function).